cfgmgr/intfmgr.cpp

```diff
@@ -25,12 +25,20 @@ IntfMgr::IntfMgr(DBConnector *cfgDb, DBConnector *appDb, DBConnector *stateDb, c
 {
 }
 
-bool IntfMgr::setIntfIp(const string &alias, const string &opCmd, const string &ipPrefixStr)
+bool IntfMgr::setIntfIp(const string &alias, const string &opCmd,
+                        const string &ipPrefixStr, const bool ipv4)
 {
     stringstream cmd;
     string res;
 
-    cmd << IP_CMD << " address " << opCmd << " " << ipPrefixStr << " dev " << alias;;
+    if (ipv4)
+    {
+        cmd << IP_CMD << " address " << opCmd << " " << ipPrefixStr << " dev " << alias;;
+    }
+    else
+    {
+        cmd <<IP_CMD << " -6 address " << opCmd << " " << ipPrefixStr << " dev " << alias;;
```

Comment:
- Add space between `<<` and `IP_CMD`
- Remove extra trailing semicolon.

Reply:
resolved